perffile/format.go

```diff
@@ -1421,6 +1422,17 @@ const (
 	DataSrcLevelNumNA       DataSrcLevelNum = 0x0f // N/A
 )
 
+type DataSrcBlock int
+
+//go:generate go run ../cmd/bitstringer/main.go -type=DataSrcBlock -strip=DataSrcBlock
+
+const (
+	DataSrcBlockData DataSrcBlock = 1 << iota
```

Comment:
DataSrcBlockData // Data could not be forwarded
DataSrcBlockAddr // Address conflict